src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/model/MethodMissingWrapperWhitelist.java

```diff
@@ -41,7 +41,9 @@
 public class MethodMissingWrapperWhitelist extends Whitelist {
     @Override
     public boolean permitsMethod(@Nonnull Method method, @Nonnull Object receiver, @Nonnull Object[] args) {
-        return (method.getName().equals("invokeMethod") || method.getName().equals("setProperty"))
+        return (method.getName().equals("invokeMethod") ||
```

Comment:
Maybe usage of static constant set and <code>contains()</code> are not that bad in this case if we're aware about performance (seems so). 🐜